### DISCUSSION TITLE
什么是loose版本？what is loose edition?

### DISCUSSION THREAD

**Discussion by scorpion202503** [electerm/electerm#2544] [Q&A]
什么是loose版？和其他版本有什么区别？
what is loose edition? what's the difference with other editions?

**Comment by zxdong262** [comment]
非loose版本会把大量的小文件打包称asar包以便减少文件数量，提高性能，loose版本保持原始文件格式，方便修改调试，对于用户而言可以无视loose版本

**Reply by scorpion202503** [reply]
我遇到一个问题，我下载了windows-x64.tar.gz的包，在家里的windows10上打开可以运行，但是放到我公司内网云桌面里面用(win7 x64)，上来就挂了，弹出一个框框提示“electerm已停止运行“。是我电脑缺什么基础包么？还是因为不能连外网导致的？有日志可以看看么？

**Reply by zxdong262** [reply]
可能是在win7运行会有一些问题

**Reply by scorpion202503** [reply]
呃……无解？
